z3/include/z3_solver.h

```diff
@@ -1,23 +1,39 @@
-#ifndef SMT_CVC4_SOLVER_H
-#define SMT_CVC4_SOLVER_H
+/*********************                                                        */
+/*! \file z3_solver.h
+** \verbatim
+** Top contributors (to current version):
+**   Amalee Wilson, Makai Mann
```

Comment:
```suggestion
**   Lindsey Stuntz, Amalee Wilson
```

Reply:
Oops, this is a copy problem I introduced. Should really just be Lindsey, since all I did was copy and paste some Yices stuff.

Reply:
Makes sense -- I figured. It's not important, but thought I'd point it out for credit purposes / to make it clear who to contact for questions.